dbt_platform_helper/commands/conduit.py

```diff
@@ -351,7 +352,7 @@ def update_conduit_stack_resources(
         DeletionPolicy: Retain
         Properties:
           RoleArn: {log_filter_role_arn}
-          LogGroupName: /copilot/{task_name}
+          LogGroupName: /copilot/{task_name}/{access}
```

Comment:
I think the `FilterName` below needs the access type, not the `LogGroupName`, but not harm with it being in the log group name as well.